.travis.yml

```diff
@@ -2,12 +2,10 @@ language: python
 python:
 - "2.7"
 # install dependency
-# install: "pip install -r requirements.txt"
-install: "echo 'nothing to install for now.'"
-# run tests
-# script: nosetests
+install: pip install -r requirements.txt
 script: "echo 'nothing to test for now.'"
```

Comment:
Please remove this line because we have test now (once this PR be merged), thanks.